modules/offers/client/components/OfferLocation.component.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OfferLocationPresentational from './OfferLocationPresentational';
+
+export default class OfferLocation extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      zoom: 13,
+      windowWidth: 0
+    };
+
+    this.handleChangeZoom = this.handleChangeZoom.bind(this);
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+  }
+
+  /* updating windows size with https://stackoverflow.com/a/42141641 */
+  componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.updateWindowDimensions);
+  }
+
+  updateWindowDimensions() {
+    this.setState(() => ({ windowWidth: window.innerWidth }));
```

Comment:
What do you think about adding [`_.throttle`](https://lodash.com/docs/4.17.11#throttle) to this so that we don't end up resizing more than maybe once every 200-500ms?

Reply:
added in 0ec4e507a9e258e0cad0b481a421c7607ef6a07f